pages/en-us/guide/server-render.mdx

```diff
@@ -15,21 +15,20 @@ export const meta = {
 All Components of `@zeit-ui/react` are compatible with **Server Render**. In fact, the document you see now is rendered by the server.
 
 <Note label="readme">
-  You just need <Link href="https://en.wikipedia.org/wiki/Single-page_application" rel="nofollow" underline>a SPA application</Link>?
-  It is recommended to skip this section.
+  If you are looking to build a SPA <Link href="https://en.wikipedia.org/wiki/Single-page_application" rel="nofollow" underline>(single page application)</Link>,
+  please feel free to skip this section.
 </Note>
 
 <Spacer y={.5} />
 
-Will using server-side rendering make my application lose the advantages of SPA? Maybe you can use the `hybrid render` application,
-Read <Link color href="https://nextjs.org/blog/next-9-3#next-gen-static-site-generation-ssg-support" rel="nofollow">
+If you are concerned about losing the advantages of a Single Page Application by implementing server-side rendering, you can use the `hybrid render` application.
+Plese read <Link color href="https://nextjs.org/blog/next-9-3#next-gen-static-site-generation-ssg-support" rel="nofollow">
 the post from the Next.js team</Link> to learn more.
 
 <Text>In addition, for <Text b>server-side render</Text> and web applications,
 we strongly recommend that you read this famous post
```

Comment:
Spaces between `post` and `7-` are ignored after compilation.

Reply:
You can see the difference here: [preview](https://react-q2fkaviap.vercel.app/en-us/guide/server-render)

Reply:
Fixed